Source/Core/Core/ReliableUDPManager/ReliableUDPConnection.h

```diff
@@ -0,0 +1,122 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <SFML/Network.hpp>
+#include "Common/CommonTypes.h"
+#include "Common/Timer.h"
+#include <queue>
+#include <map>
+#include <memory>
+#include "Common/Thread.h"
+
+class ReliableUDPConnection
+{
+#define udpBitType	u32
+#define udpBitMax	UINT32_MAX
```

Comment:
This should be upper-cased since it's a constant: `UDP_BIT_MAX`. As well as being a concrete type (so, a u32 in this case)